src-web/components/modals/RemoveResourceModal.js

```diff
@@ -149,8 +149,9 @@ class RemoveResourceModal extends React.Component {
             {reqStatus === REQUEST_STATUS.ERROR &&
               <AcmAlert
                 variant='danger'
-                title=''
-                subtitle={reqErrorMsg || msgs.get('error.default.description', locale)} />}
+                isInline={true}
+                noClose={true}
```

Comment:
I tried fixing this, too. I had to add both:
```
noClose={true}
canClose={false}
```

Reply:
doesn't look like AcmAlert have props `canClose` https://github.com/open-cluster-management/ui-components/blob/main/src/AcmAlert/AcmAlert.tsx#L102-L110

Reply:
Yup--nevermind. I must have been confusing them--it works.